slurmstepd: error: Unable to create TMPDIR [/tmp/siw001/874536]: No such file or directory
slurmstepd: error: Setting TMPDIR to /tmp
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    raise ValueError('invalid controller')
ValueError: invalid controller
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
Traceback (most recent call last):
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    raise ValueError('invalid controller')
ValueError: invalid controller
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    raise ValueError('invalid controller')
ValueError: invalid controller
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    raise ValueError('invalid controller')
ValueError: invalid controller
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    raise ValueError('invalid controller')
ValueError: invalid controller
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    raise ValueError('invalid controller')
ValueError: invalid controller
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    raise ValueError('invalid controller')
ValueError: invalid controller
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    raise ValueError('invalid controller')
ValueError: invalid controller
    raise ValueError('invalid controller')
ValueError: invalid controller
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    raise ValueError('invalid controller')
    raise ValueError('invalid controller')
ValueError: invalid controller
ValueError: invalid controller
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    raise ValueError('invalid controller')
ValueError: invalid controller
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    raise ValueError('invalid controller')
ValueError: invalid controller
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    raise ValueError('invalid controller')
ValueError: invalid controller
    raise ValueError('invalid controller')
ValueError: invalid controller
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    raise ValueError('invalid controller')
ValueError: invalid controller
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    raise ValueError('invalid controller')
ValueError: invalid controller
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    raise ValueError('invalid controller')
ValueError: invalid controller
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    raise ValueError('invalid controller')
ValueError: invalid controller
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    raise ValueError('invalid controller')
ValueError: invalid controller
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    raise ValueError('invalid controller')
ValueError: invalid controller
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    raise ValueError('invalid controller')
ValueError: invalid controller
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    raise ValueError('invalid controller')
ValueError: invalid controller
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    raise ValueError('invalid controller')
ValueError: invalid controller
    raise ValueError('invalid controller')
ValueError: invalid controller
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    raise ValueError('invalid controller')
ValueError: invalid controller
    raise ValueError('invalid controller')
ValueError: invalid controller
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    raise ValueError('invalid controller')
ValueError: invalid controller
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    raise ValueError('invalid controller')
ValueError: invalid controller
    raise ValueError('invalid controller')
ValueError: invalid controller
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    raise ValueError('invalid controller')
ValueError: invalid controller
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    raise ValueError('invalid controller')
ValueError: invalid controller
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    raise ValueError('invalid controller')
ValueError: invalid controller
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
Traceback (most recent call last):
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
Traceback (most recent call last):
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    rank = main(args)
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    raise ValueError('invalid controller')
ValueError: invalid controller
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
Traceback (most recent call last):
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 271, in <module>
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    rank = main(args)
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    rank = main(args)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/./main_gef.py", line 80, in main
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    raise ValueError('invalid controller')
ValueError: invalid controller
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    raise ValueError('invalid controller')
ValueError: invalid controller
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    Q = stepper.step(Q, param.dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/integrator.py", line 56, in step
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    raise ValueError('invalid controller')
ValueError: invalid controller
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    raise ValueError('invalid controller')
ValueError: invalid controller
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    result = self.__step__(Q, dt)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/epi.py", line 99, in __step__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    raise ValueError('invalid controller')
ValueError: invalid controller
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    phiv, stats = exode(1., matvec_handle, vec, method=self.exode_method, controller=self.exode_controller, atol = self.tol, task1 = False, verbose=False)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/solvers/exode.py", line 38, in exode
    raise ValueError('invalid controller')
ValueError: invalid controller
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    solver = method(fun, t0, y0, tf, controller=controller, first_step=exode.first_step, rtol=rtol, atol=atol)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 69, in __init__
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    raise ValueError('invalid controller')
ValueError: invalid controller
    self._init_control(controller)
  File "/fs/hestia_Heccc/rpnatm/siw001/gef/integrators/explicit_runge_kutta.py", line 109, in _init_control
    raise ValueError('invalid controller')
ValueError: invalid controller
    raise ValueError('invalid controller')
ValueError: invalid controller
    raise ValueError('invalid controller')
ValueError: invalid controller
    raise ValueError('invalid controller')
ValueError: invalid controller
    raise ValueError('invalid controller')
ValueError: invalid controller
